Userland/js.cpp

```diff
@@ -351,6 +353,126 @@ int main(int argc, char** argv)
 
         editor = make<Line::Editor>();
         editor->initialize();
+        if (syntax_highlight)
+            editor->on_display_refresh = [](Line::Editor& editor) {
+                using namespace Line;
+                using Color = Style::Color;
```

Comment:
ew but ok

Reply:
honestly...`Line::Style::Color::Red` is a mouthful and a half...do you have any alternatives to this by chance?
`Line::Color` sounds...weird

Reply:
I'm gonna just leave it be as `Line::Style::Color` and...maybe some day someone will find a better name.